### PR TITLE
Perfomance

### PR DESCRIPTION
**Pull request by codewanderlust**
In this pull request, Ayana and I optimized some slow queries in the database.
Some of the steps and screenshots showing the performance of the queries are attached below

 -  SELECT COUNT(*) FROM visits where animal_id = 4;

**Before optimizing the query performance** 

**After optimizing the query performance** 

- SELECT * FROM visits where vet_id = 2; 
**Before optimizing the query performance** 

**After optimizing the query performance** 

- SELECT * FROM owners where email = 'owner_18327@mail.com';

**Before optimizing the query performance** 

**After optimizing the query performance**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +164,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (t-yanick) [inline README.md]: - [ ] Great job working on this milestone. When I click on your MIT license link, it redirects to an error 404 page. It not professional to push broken links to Github. Kindly fix this issue.
Screen Shot 2023-02-28 at 6 18 39 AM

File: README.md [hunk 2]
```diff
@@ -100,11 +60,7 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **A database that allows for querying and filtering based on it's features.**
```
Comment (t-yanick) [inline README.md]: - [ ] At the moment, your Live demo link is broken. You can write the coming soon text simply without making it a link. Kindly fix this

**Review by t-yanick — COMMENTED** [review]
Hi @RoyJumah and @atatm , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work. Commits from both team members
:heavy_check_mark: Screenshots for all queries attached to PR description

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @RoyJumah 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**